R in Goverment Community Call (#653)
ropensci/roweb3: content/commcalls/2023-10-31/index.md

---
title: "R in Goverment"
dateStart: 2023-10-31T16:00:00 # UTC!!
dateEnd: 2023-10-31T17:00:00 # UTC!!
date: 2023-10-31T17:00:00 # UTC!! same as dateEnd
publishDate: 2023-09-30
description: 
author:
  - Luíza Andrade
  - Karly Harker
  - Ahmadou Dicko
  - Pablo Tiscornia
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: oct2023-goverment
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:

speakers:  
  - name: Luíza Andrade

>     bio: ""

Author: ```suggestion
    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
```

    img: luisa-andrade.png
  - name: Karly Harker
    bio: ""
    img: karly-harker.jpg
  - name: Ahmadou Dicko

>     bio: ""

Author: ```suggestion
    bio: "I am a statistician and R enthusiast. I believe in the use of data science for social good and have worked with several non-profit and research organizations."
```

>     bio: "I am a statistician and R enthusiast. I believe in the use of data science for social good and have worked with several non-profit and research organizations."

Author: ```suggestion
    bio: "Ahmadou Dicko is a humanitarian data scientist, currently serving as a Statistics and Data Analysis officer at UNHCR regional bureau for West and Central Africa. He is a fervent advocate for reproducibility in the humanitarian sector and has built several R packages and tools to streamline data processes. With M.Sc. in Statistics and a Ph.D. in Climate Change Economics, Ahmadou combines his profound knowledge of economics and data science to drive transformative change in the humanitarian sector. He is the founder and co-organizer of the Dakar R User Group and a recognized leader in promoting data literacy and community building around humanitarian data science."
```

>     img: luisa-andrade.png

Author: ```suggestion
    img: img/community/luiza-andrade.jpg
```

>     img: img/community/luiza-andrade.jpg

Author: ```suggestion
    img: /img/community/luiza-andrade.jpg
```

>     bio: ""

Author: ```suggestion
    bio: "Environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"
```

>     img: karly-harker.jpg

Author: ```suggestion
    img: /img/community/karly-harker.jpeg
```

>     img: ahmadou-dicko.jpg

Author: ```suggestion
    img: img/community/ahmadou-dicko.jpg
```

>     img: img/community/ahmadou-dicko.jpg

Author: ```suggestion
    img: /img/community/ahmadou-dicko.jpg
```

  - name: Pablo Tiscornia
    bio: ""

>     img: pablo-tiscornia.jpg

Author: ```suggestion
    img: img/community/pablo-tiscornia.jpeg
```

>     img: img/community/pablo-tiscornia.jpeg

Author: ```suggestion
    img: /img/community/pablo-tiscornia.jpeg
```

>     bio: ""

Author: ```suggestion
    bio: "Pablo is a graduate in Sociology from the University of Buenos Aires, Argentina. Currently, he works at C40 Cities as a Data Scientist, with a focus on improving data workflows related to Climate Change. He also serves as an instructor for Data Processing with R for Social Sciences at several National Universities in Argentina.
He has made significant contributions to the development of R as a co-author of various packages used by Argentina's leading official statistical offices (INDEC, MINTURyDEP, and DGEyC). He has also been a co-organizer of the R community in Buenos Aires."
```

---
title: "R in Government"
dateStart: 2023-10-31T16:00:00 # UTC!!
dateEnd: 2023-10-31T17:00:00 # UTC!!
date: 2023-10-31T17:00:00 # UTC!! same as dateEnd
publishDate: 2023-09-30
description: 
author:
  - Luíza Andrade
  - Karly Harker
  - Ahmadou Dicko
  - Pablo Tiscornia
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome

> slug: oct2023-goverment

Author: ```suggestion
slug: oct2023-government
```

notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:

speakers:  
  - name: Luíza Andrade
    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
    img: /img/community/luiza-andrade.jpg
  - name: Karly Harker

>     bio: "Environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"

Reviewer: could this one be sentences too?

Author: Unfortunately I only have that information.

>     bio: "Environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"

Author: ```suggestion
    bio: "Karly Harker is an environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"
```

    img: /img/community/ahmadou-dicko.jpg
  - name: Pablo Tiscornia
    bio: "Pablo is a graduate in Sociology from the University of Buenos Aires, Argentina. Currently, he works at C40 Cities as a Data Scientist, with a focus on improving data workflows related to Climate Change. He also serves as an instructor for Data Processing with R for Social Sciences at several National Universities in Argentina.
He has made significant contributions to the development of R as a co-author of various packages used by Argentina's leading official statistical offices (INDEC, MINTURyDEP, and DGEyC). He has also been a co-organizer of the R community in Buenos Aires."
    img: /img/community/pablo-tiscornia.jpeg
tags:
  - community call
  - events
  - goverment
deets: |

>     Meeting ID:  

Author: ```suggestion
    Meeting ID:  831 0029 0068 
```

>     Passcode: 

Author: ```suggestion
    Passcode: 042189
```

> zoomurl: 

Author: ```suggestion
zoomurl: https://numfocus-org.zoom.us/j/83100290068?pwd=c4bloSdA1zNaQ7IzJD83BHD3I54zab.1
```

---

> In this community call, our panelist will share their experiences and examples of projects with R at different levels of government and in different countries.  We invite you to learn about the challenges and lessons learned from our panelists and attendees in their efforts to make their government data, processes, and analysis more open and reproducible.    

Reviewer: ```suggestion
In this community call, our panelists will share their experiences and examples of projects with R at different levels of government and in different countries.  
We invite you to learn about the challenges and lessons learned from our panelists and attendees in their efforts to make their government data, processes, and analyses more open and reproducible.    
```

---
title: "R in Government"
dateStart: 2023-10-31T16:00:00 # UTC!!
dateEnd: 2023-10-31T17:00:00 # UTC!!
date: 2023-10-31T17:00:00 # UTC!! same as dateEnd
publishDate: 2023-09-30
description: 
author:
  - Luíza Andrade
  - Karly Harker
  - Ahmadou Dicko
  - Pablo Tiscornia
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: oct2023-government
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:

speakers:  
  - name: Luíza Andrade
    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
    img: /img/community/luiza-andrade.jpg
  - name: Karly Harker

>     bio: "Karly Harker is an environmental Data Science Specialist at the British Columbia Ministry of Environment and Climate Change. Karly is an Environment and Resource Studies and Biology graduate from the University of Waterloo.

Author: ```suggestion
    bio: "Karly Harker is an environmental Data Science Specialist at the British Columbia Ministry of Environment and Climate Change. Karly is an Environment and Resource Studies and Biology graduate from the University of Waterloo."
```
